Formula/bookloupe.rb

```diff
@@ -0,0 +1,62 @@
+class Bookloupe < Formula
+  desc "Lists common formatting errors in a Project Gutenberg candidate file"
```

Comment:
Tiny nitpick, but I think it reads better not to use the 3<sup>rd</sup> person singular here, i.e. use instead:

```ruby
desc "List common formatting errors in a Project Gutenberg candidate file"
```